flink-runtime/src/test/java/org/apache/flink/runtime/failure/FailureEnricherUtilsTest.java

```diff
@@ -109,6 +109,15 @@ public void testGetFailureEnrichers() {
         assertThat(enrichers).hasSize(1);
         // verify that the failure enricher was created and returned
         assertThat(enrichers.iterator().next()).isInstanceOf(TestEnricher.class);
+
+        // Valid plus Invalid Name combination
+        configuration.set(
+                JobManagerOptions.FAILURE_ENRICHERS_LIST,
+                FailureEnricherUtilsTest.class.getName() + "," + TestEnricher.class.getName());
+        final Collection<FailureEnricher> validInvalidEnrichers =
+                FailureEnricherUtils.getFailureEnrichers(configuration, createPluginManager());
+        assertThat(validInvalidEnrichers).hasSize(1);
+        assertThat(enrichers.iterator().next()).isInstanceOf(TestEnricher.class);
```

Comment:
It is recommended to verify this way to make it clearer.

```suggestion
        assertThat(validInvalidEnrichers)
                .satisfiesExactly(
                        enricher -> assertThat(enricher).isInstanceOf(TestEnricher.class));
```

Reply:
It's appreciated if you could update line 109~111.